{"name":"A. Case of the Zeros and Ones","group":"Codeforces - Remote contest #30","url":"https://codeforces.com/gym/455594/problem/0","interactive":false,"memoryLimit":256,"timeLimit":1000,"tests":[{"input":"4\n1100\n","output":"0\n","id":1689421207669},{"input":"5\n01010\n","output":"1\n","id":1689421207698},{"input":"8\n11101111\n","output":"6\n","id":1689421207663}],"testType":"single","input":{"type":"stdin"},"output":{"type":"stdout"},"languages":{"java":{"mainClass":"Main","taskClass":"ACaseOfTheZerosAndOnes"}},"batch":{"id":"80800b80-bf1e-4571-8fb6-c234bcd214e3","size":1},"srcPath":"d:\\Fasika\\vs code python\\Code_force-compitative-programming\\A_Case_of_the_Zeros_and_Ones.py"}